openc3/lib/openc3/microservices/periodic_microservice.rb

```diff
@@ -18,15 +18,41 @@
 
 require 'openc3/microservices/microservice'
 require 'openc3/models/offline_access_model'
+require 'openc3/models/news_model'
+# The VERSION and ENTERPRISE constants are set by settings_api.rb
+require 'openc3/api/settings_api'
 
 module OpenC3
+  class LocalApi
+    include Api
+  end
```

Comment:
Why not just include Api in the microservice class?

Reply:
I didn't want to pollute the entire class just to call the one method but I guess it doesn't matter